buildmaster/H1_1JET_319GEV_290PB-1_DIF/metadata.yaml

```diff
@@ -0,0 +1,120 @@
+setname: 'H1_1JET_319GEV_290PB-1_DIF'
+
+nnpdf_metadata:
+  nnpdf31_process: "1JET"
```

Comment:
As per https://github.com/NNPDF/nnpdf/pull/1825#issuecomment-1786778365 please change all 1JET / DIS_JET to HERAJET
and 2JET / DIS_DIJET to HERADIJET.

(this is also nice in that we will have the nnpdf31_process agree with the process_type for a change)

Reply:
done